src/WebJobs.Extensions.DurableTask/FunctionFailedException.cs

```diff
@@ -21,12 +21,21 @@ internal FunctionFailedException()
         {
         }
 
-        internal FunctionFailedException(string message)
+        /// <summary>
+        /// Initializes a new instance of a <see cref="FunctionFailedException"/>.
+        /// </summary>
+        /// <param name="message">A message describing where to look for more details.</param>
+        public FunctionFailedException(string message)
```

Comment:
Unsure if this method really should be public. It isn't currently used in our code (but the `[Serializable]` attribute might require the method exists).  Once we expose it as public, we can't remove it without it being a breaking change.

Reply:
It's pretty common to have exception types with public constructors like this. I think it's fine to make this public.